### PR TITLE
Add safety_limiter_msgs package

### PR DESCRIPTION
**Pull request by DaikiMaekawa**

### CODE REVIEW

**Comment by at-wat** [comment]
## [#39] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by DaikiMaekawa** [comment]
@at-wat PTAL

**Comment by at-wat** [comment]
## [#40] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [#41] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by DaikiMaekawa** [comment]
@at-wat Added. https://github.com/at-wat/neonavigation_msgs/pull/16/files#diff-890558fa1cd6f908ee9876a9aecbdc8fR9-R11

**Comment by at-wat** [comment]
@DaikiMaekawa basically looks good to me. Let's merge it after implementing a publisher in safety_limiter.

**Comment by at-wat** [comment]
## [#42] FAILED on kinetic

```catkin_make``` failed

**Comment by at-wat** [comment]
## [#43] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: safety_limiter_msgs/msg/SafetyLimiterStatus.msg [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+int8 COLLISION_IS_PREDICTED = 1
+int8 WATCHDOG_TIMED_OUT = 3
+
+Header header
+int8 status
+
+float64 limit_ratio
+bool is_cloud_available
+bool has_watchdog_timed_out
+time stuck_started_since
```
Comment (at-wat) [inline safety_limiter_msgs/msg/SafetyLimiterStatus.msg]: meaning of zero value (not being stuck) should be described as a comment.

**Comment by at-wat** [comment]
## [#44] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: safety_limiter_msgs/package.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>safety_limiter_msgs</name>
+  <version>0.0.0</version>
```
Comment (at-wat) [inline safety_limiter_msgs/package.xml]: should be 0.3.1

Comment (DaikiMaekawa) [reply safety_limiter_msgs/package.xml]: done

File: safety_limiter_msgs/msg/SafetyLimiterStatus.msg (hunk 1 above)
Comment (DaikiMaekawa) [reply safety_limiter_msgs/msg/SafetyLimiterStatus.msg]: added

**Comment by at-wat** [comment]
## [#45] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
@DaikiMaekawa dependency from `neonavigation/package.xml` should be added. https://github.com/at-wat/neonavigation_msgs/pull/16#pullrequestreview-288989676

**Comment by at-wat** [comment]
## [#46] PASSED on kinetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_convert.xml: 2 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 3 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
merging

**Pull request merged**